src/components/General/Navbar.jsx

```diff
@@ -7,8 +7,8 @@ import HackLogo from '../../images/hothXI-logo.svg';
 export default function Navbar() {
 	const [isOpen, setIsOpen] = useState(false);
 	const [isMobile, setIsMobile] = useState(false);
-	const IsApplyActive = false;
-	const ApplyButtonClassName = 'apply-btn ' + (IsApplyActive?'apply-active':'apply-inactive');
+	const isApplyActive = true;
```

Comment:
Please change this to false. Before rerequesting, make sure that all previous feedback is still fixed.